activerecord/test/cases/adapters/postgresql/geometric_test.rb

```diff
@@ -212,10 +217,12 @@ def test_alternative_format
       :a_polygon      => '2.0, 3, 5.5, 7.0, 8.5, 11.0',
       :a_circle       => '((5.3, 10.4), 2)'
     )
+    g[:a_line] = '(2.0, 3), (4.0, 6.0)' if supports_line?
 
     g.save!
 
     h = PostgresqlGeometric.find(g.id)
+    assert_equal '{1.5,-1,0}', h.a_line if supports_line?
```

Comment:
Can we put stuff about the `line` type in a separate test? I don't like how `supports_line?` is spread all over this test.